Commons/src/main/java/com/github/jsonldjava/core/Constants.java

```diff
@@ -75,18 +75,21 @@ public class Constants {
 		allowedScalars.put(AppConstants.SUBSCRIPTION_UPDATE_PAYLOAD, NGSIConstants.NGSI_LD_SCOPE_Q);
 
 		allowedScalars.put(AppConstants.ENTITY_CREATE_PAYLOAD, NGSIConstants.NGSI_LD_HAS_OBJECT);
+		allowedScalars.put(AppConstants.ENTITY_CREATE_PAYLOAD, "https://uri.etsi.org/ngsi-ld/default-context/objectType");
```

Comment:
Please add this "objectType" uri in NGSIConstants class.

Reply:
done